ui/narrative/methods/align_reads_using_bowtie2/display.yaml

```diff
@@ -17,7 +17,7 @@ method-suggestions :
     related :
         []
     next :
-        [assemble_transcripts_using_cufflinks]
+        [assemble_transcripts_using_cufflinks,assemble_transcripts_with_stringtie]
```

Comment:
Should be `kb_cufflinks.assemble_transcripts_using_cufflinks, kb_stringtie.run_stringtie`